docs/Workflow.md

```diff
@@ -77,11 +77,15 @@ You can now edit the code on the `myfeature` branch.
 #### Run cmake & make
 
 Follow the directions to build ONNX-MLIR in this section of the main 
-[README](../README.md#onnx-mlir-this-project)
+[README](../README.md#onnx-mlir-this-project).
 
+We expect code to compile without generating any compiler warnings. 
 
 #### Run Test
 
+In general, the new features must be tested in one or more of our test suite. 
+At a high level, our testing strategy includes  `literal` tests (`check-onnx-lit` below), end-to-end tests derived from the ONNX Standard (`check-onnx-backend` and derivatives below, and semi-exhaustive numerical tests (`test` below).
```

Comment:
extra white space: includes  `literal` --> includes `literal`

Reply:
Removed, my understanding is that multiple spaces are ignored

> Markdown is used primarily to generate HTML, and HTML collapses white spaces by default. Use &nbsp; instead of space characters.

Reply:
Oh I did not know they are. Good info.